frontend/packages/console-shared/src/components/dashboard/status-card/states.tsx

```diff
@@ -20,83 +20,64 @@ export enum HealthState {
 }
 
 export const healthStateMapping: { [key in HealthStateMappingKeys]: HealthStateMappingValues } = {
-  [HealthState.OK]: {
-    icon: <GreenCheckCircleIcon />,
-  },
-  [HealthState.ERROR]: {
-    icon: <RedExclamationCircleIcon />,
-    message: 'Degraded',
-  },
-  [HealthState.WARNING]: {
-    icon: <YellowExclamationTriangleIcon />,
-    message: 'Degraded',
-  },
-  [HealthState.UPDATING]: {
-    icon: <BlueSyncIcon />,
-    message: 'Updating',
-  },
-  [HealthState.PROGRESS]: {
-    icon: <InProgressIcon />,
-    message: 'Pending',
-  },
-  [HealthState.UNKNOWN]: {
-    icon: <GrayUnknownIcon />,
-    message: 'Unknown',
-  },
-  [HealthState.NOT_AVAILABLE]: {
-    icon: <GrayUnknownIcon />,
-    message: 'Not available',
-  },
-};
-
-export const healthPriority: {
-  [key in HealthStateMappingKeys]: PriorityHealthState;
-} = {
   [HealthState.OK]: {
     priority: 0,
     health: HealthState.OK,
-    ...healthStateMapping[HealthState.OK],
+    icon: <GreenCheckCircleIcon />,
   },
   [HealthState.UNKNOWN]: {
     priority: 1,
     health: HealthState.UNKNOWN,
-    ...healthStateMapping[HealthState.UNKNOWN],
+    icon: <GrayUnknownIcon />,
+    message: 'Unknown',
   },
   [HealthState.PROGRESS]: {
     priority: 2,
     health: HealthState.PROGRESS,
-    ...healthStateMapping[HealthState.PROGRESS],
+    icon: <InProgressIcon />,
+    message: 'Pending',
   },
   [HealthState.UPDATING]: {
     priority: 3,
     health: HealthState.UPDATING,
-    ...healthStateMapping[HealthState.UPDATING],
+    icon: <BlueSyncIcon />,
+    message: 'Updating',
   },
   [HealthState.WARNING]: {
     priority: 4,
     health: HealthState.WARNING,
-    ...healthStateMapping[HealthState.WARNING],
+    icon: <YellowExclamationTriangleIcon />,
+    message: 'Degraded',
   },
   [HealthState.ERROR]: {
     priority: 5,
     health: HealthState.ERROR,
-    ...healthStateMapping[HealthState.ERROR],
+    icon: <RedExclamationCircleIcon />,
+    message: 'Degraded',
   },
-  [HealthState.NOT_AVAILABLE]: {
+  [HealthState.LOADING]: {
     priority: 6,
+    health: HealthState.LOADING,
+    icon: <div className="skeleton-health" />,
+    message: 'Loading',
+  },
+  [HealthState.NOT_AVAILABLE]: {
+    priority: 7,
     health: HealthState.NOT_AVAILABLE,
-    ...healthStateMapping[HealthState.NOT_AVAILABLE],
+    icon: <GrayUnknownIcon />,
+    message: 'Not available',
   },
 };
 
+type HealthStateMappingKeys = keyof typeof HealthState;
 export type PriorityHealthState = {
```

Comment:
seems like we dont need this one anymore as its the same as `HealthStateMappingValues`